### ISSUE TITLE
TypeError Window.SetCursor

### ISSUE THREAD

**Issue by kdschlosser** [wxWidgets/Phoenix#1309]
This would be more of a suggestion then an issue. This is the first time I have played around with the cursors. and the wx.CURSOR_* constants do not seem to be consistent with the majority of the other constants..

examples are 

* wx.RED: wx.Colour(255, 0, 0, 255)
* wx.TRANSPARENT_BRUSH: wx.Brush(wx.TransparentColour)

the wx.CURSOR_* constants are enumeration values instead of being wx.Cursor instances. I do not know if maybe there is a reason as to why.. If there is a reason if you would care to enlighten me I would appreciate it.. I am always trying to learn. and if there isn't then it is worth mentioning because of the breaking changes already made in the API between 3.X and 4.X

**Comment by RobinD42** [comment]
It's simply because it's an enum in C++, rather than some global instances of cursor objects. In Classic we created simple integer values in Python for the C++ enums, but now actual enum classes are being generated. This should be considered a step up.

Unless I'm missing something there really isn't any significant change in the API since Classic. `wx.Window.SetCursor` still requires an actual  `wx.Cursor` object. `wx.StockCursor` still exists, although it's deprecated. A `wx.Cursor` can still be created using either an enum element or a plain integer.

**Comment by kdschlosser** [comment]
I know it is the same as the classic API. I just just figured it may be worth mentioning because it is not in line with the rest of the constants. an example would be the EVT_* constants which are instances as well as the various colors, fonts, pens, brushes, there are also 3 constants that are wx.Cursor instances.

It just doesn't seem to conform to what appears to be a standard way that you have set up the rest of the library. And because of the major version bump and other API breaking changes that have been made it would not be so much of an issue to make the CURSOR_* constants work the same.

I do not know if there would be a need to know the enumeration value for the cursors. If there is then the names of the constants for the enumeration values would be wxCURSOR_* I would imagine

I just noticed it because I wanted to change the cursor and I had not used them in a while and I just passed the constant to SetCursor not remembering that I had to pass the constant to wx.Cursor first.
